integration/survey-api.md

```diff
@@ -232,22 +232,22 @@ Before showing examples working with JSON data, it is necessary to understand th
 ```json
 [
     {
-        "metaData": {"submission_id": 1, "timestamp": "2020-10-13T10:15:26.388573"},
+        "metadata": {"submission_id": 1, "created": "2020-10-13T10:15:26.388573"},
```

Comment:
the example data here is outdated in other ways too, e.g. `data` -> `answers`, and it is no longer an array, I'll address that in another PR :)